flocker/control/_diffing.py

```diff
@@ -0,0 +1,247 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+# -*- test-case-name: flocker.control.test.test_diffing -*-
+
+from pyrsistent import (
+    PClass,
+    PMap,
+    PVector,
+    PSet,
+    field,
+    freeze,
+    pvector,
+)
+
+from zope.interface import Interface, implementer
+
+
+class _DiffChange(Interface):
+    """
+    Interface for a diff change.
+
+    This is simply something that can be applied to an object to create a new
+    object.
+
+    This interface is created more to simply documentation than for any of the
+    actual zope.interface mechanisms.
+    """
```

Comment:
Typo "more to simply documentation"
And maybe drop the word "simply" from the line above while you're here.